lib/erblint-github/linters/github/accessibility/avoid_generic_link_text_counter.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
+
+require_relative "../../custom_helpers"
+
+module ERBLint
+  module Linters
+    module GitHub
+      module Accessibility
+        class AvoidGenericLinkTextCounter < Linter
+          include ERBLint::Linters::CustomHelpers
+          include LinterRegistry
+
+          BANNED_GENERIC_TEXT = [
```

Comment:
Is there anything to remove from here? Anything else we should add in here?

Reply:
This is amazing 🤩 Thank you for doing this! What do you think about adding "Here" as well? I've seen text such as `Read more about it <a href="#">here</a>

Reply:
Great idea!!! <3

Reply:
Updated in: https://github.com/github/erblint-github/pull/32/commits/80ebd6bbbba39dd2dce94b1ce32038837bd288e8 ❤️